src/scss/cdb-components/tags.scss

```diff
@@ -0,0 +1,48 @@
+// Typography styles
+// ----------------------------------------------
+
+/* SG
+# Tags
+
+```
+<button class="CDB-Tag is-public CDB-Text is-semibold CDB-Size-small u-iBlock">
+  PUBLIC
+</button>
+<button class="CDB-Tag is-link CDB-Text is-semibold CDB-Size-small u-iBlock">
+  LINK
+</button>
+<button class="CDB-Tag is-password CDB-Text is-semibold CDB-Size-small u-iBlock">
+  PASSWORD
+</button>
+<button class="CDB-Tag is-private CDB-Text is-semibold CDB-Size-small u-iBlock">
+  PRIVATE
+</button>
+```
+*/
+
+@import '../cdb-variables/sizes';
+@import '../cdb-variables/colors';
+
+
+.CDB-Tag {
+  border: 1px solid red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Properties should be ordered padding, border, border-radius